tasks/dataverse-prereqs.yml

```diff
@@ -84,3 +84,11 @@
   when:
     - ansible_os_family == "Debian"
   tags: prereqs
+
+- name: Dataverse installer calls /usr/bin/python explicitly
```

Comment:
Why? Which version of Python is /usr/bin/python?

Reply:
RHEL/CentOS8 ship only with python3, as /usr/bin/python3. This alternatives step merely symlinks /usr/bin/python to /usr/bin/python3 for benefit of install.py